src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/BuildTest.cs

```diff
@@ -397,14 +397,25 @@ public void BuildMultiDexApplication (bool useJackAndJill, string fxVersion)
 		[Test]
 		public void MultiDexCustomMainDexFileList ()
 		{
+			var expected = @"android/support/multidex/ZipUtil$CentralDirectory.class
+android/support/multidex/MultiDexApplication.class
+android/support/multidex/MultiDex$V19.class
+android/support/multidex/MultiDex$V4.class
+android/support/multidex/ZipUtil.class
+android/support/multidex/MultiDexExtractor$1.class
+android/support/multidex/MultiDexExtractor.class
+android/support/multidex/MultiDex$V14.class
+android/support/multidex/MultiDex.class
+MyTest
+";
 			var proj = CreateMultiDexRequiredApplication ();
 			proj.SetProperty ("AndroidEnableMultiDex", "True");
 			proj.OtherBuildItems.Add (new BuildItem ("MultiDexMainDexList", "mymultidex.keep") { TextContent = () => "MyTest", Encoding = Encoding.ASCII });
 			proj.OtherBuildItems.Add (new BuildItem ("AndroidJavaSource", "MyTest.java") { TextContent = () => "public class MyTest {}", Encoding = Encoding.ASCII });
 			var b = CreateApkBuilder ("temp/MultiDexCustomMainDexFileList");
 			b.ThrowOnBuildFailure = false;
 			Assert.IsTrue (b.Build (proj), "build should succeed. Run will fail.");
-			Assert.AreEqual ("MyTest", File.ReadAllText (Path.Combine (Root, b.ProjectDirectory, proj.IntermediateOutputPath, "multidex.keep")), "unexpected multidex.keep content");
+			Assert.AreEqual (expected, File.ReadAllText (Path.Combine (Root, b.ProjectDirectory, proj.IntermediateOutputPath, "multidex.keep")), "unexpected multidex.keep content");
```

Comment:
Do we need to worry about end-of-line characters here, particularly when this test is eventually run on Windows? Should we normalize?

@jonathanpeppers: Thoughts?

Reply:
I think it should compare in a way that the line endings won't matter. [Here](https://github.com/xamarin/java.interop/blob/5b18729d97b1e642ceb2c6d39e2e76ff76c188b2/tools/generator/Tests/BaseGeneratorTest.cs#L95) is an example in Java.Interop that works pretty well.